pkg/metrics/metrics.go

```diff
@@ -71,6 +71,12 @@ var (
 	// LabelDatapathFamily marks which protocol family (IPv4, IPV6) the metric is related to.
 	LabelDatapathFamily = "family"
 
+	// LabelStatus the label from completed task
+	LabelStatus = "status"
+
+	// LabelScope is the label used to defined multiples scopes in the same metric
```

Comment:
Perhaps, add "For example, one counter may measure a metric over the scope of the entire event (scope=global), or just part of an event (scope=slow_path)"?